strings/base_activation.h

```diff
@@ -79,7 +87,7 @@ namespace winrt::impl
                 continue;
             }
 
-            if constexpr (isSameInterfaceAsIActivationFactory)
+            if (isSameInterfaceAsIActivationFactory)
```

Comment:
Don't need isSameInterfaceAsIActivationFactory in the helper function. That was an optimization to allow multiple instances of get_runtime_activation_factory_impl to get folded together if isSameInterfaceAsIActivationFactory ==true, but that doesn't apply to the function pointer callout. Can just use the "false" case in the function pointer callout.

Reply:
In that case should I remove the template from `get_runtime_activation_factory_impl` entirely?  It wouldn't change any behavior at that point.